internals/cli/masker/matcher_test.go

```diff
@@ -14,6 +18,8 @@ func TestMultipleMatcher(t *testing.T) {
 		[]byte("test"),
 		[]byte("test but longer"),
 		[]byte("another first"),
+		[]byte("112"),
+		[]byte("22222222223"),
```

Comment:
how does it work for repeating repetitions, e.g. `111211121112`?

Reply:
Added a case.